### PR TITLE
Add support for multiple windows with lwjgl

### PR DESCRIPTION
**Pull request by exuvo**

### CODE REVIEW

**Comment by exuvo** [comment]
Made vulkan when cases consistent to TODO().
Changed window and implGl3 var to val.

**Comment by exuvo** [comment]
Should I press the "merge pull request" button? Seems like something the approver should do no?

**Comment by Sylvyrfysh** [comment]
You or I could merge it, but I usually defer merging PR's like this to elect. I'll review and try to pick out stuff, but I don't do enough around here to feel confident merging large/breaking things into these code bases

**Comment by Sylvyrfysh** [comment]
On a side note, this brings up the question again of dropping JOGL. Since we haven't had many people use it, we can either drop it, or it needs to be updated as well.

**Comment by Sylvyrfysh** [comment]
Though I will note that those tests ran fine anyways

File: src/main/kotlin/imgui/impl/LwjglGlfw.kt [hunk 1]
```diff
@@ -215,42 +260,14 @@ object LwjglGlfw {
...
-
     fun shutdown() {
 
         // Destroy GLFW mouse cursors
         mouseCursors.forEach(::glfwDestroyCursor)
         mouseCursors.fill(NULL)
 
         when (clientApi) {
-            GlfwClientApi.OpenGL -> ImplGL3.destroyDeviceObjects()
+            GlfwClientApi.OpenGL -> implGl3?.destroyDeviceObjects()
```
[the remark this answers is not in the document]
Comment (elect86) [reply src/main/kotlin/imgui/impl/LwjglGlfw.kt]: Agree

**Comment by elect86** [comment]
Regarding JOGL, I'll drop it soon in the next releases, we still rely on it @work, I'll take care of update it

Thanks all, nice work guys

**Pull request merged**